cmd/downloadStunnerCtl.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# stunnerctl download script
+#
+# inspired by https://raw.githubusercontent.com/istio/istio/master/release/downloadIstioCtl.sh
+
+
+REPO=https://github.com/l7mp/stunner
+
+# Determine OS
+OS="${TARGET_OS:-$(uname)}"
+if [ "${OS}" = "Darwin" ] ; then
+  OSEXT="darwin"
+else
+  OSEXT="linux"
+fi
+
+# Determine the latest STUNner version
+if [ "${STUNNER_VERSION}" = "" ] ; then
+  STUNNER_VERSION="$(curl -sL ${REPO}/releases | \
```

Comment:
This is clever! I'm wondering why there's no GitHub API for this. Or is there?

Reply:
Good point, a  "get latest release of this repo" API call indeed exists: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release  since 2022. I gues Istio devs made their scripts prior to the API release.

Reply:
Done in 0d9b7ec